planner/core/rule_column_pruning.go

```diff
@@ -34,7 +35,7 @@ func getUsedList(usedCols []*expression.Column, schema *expression.Schema) []boo
 	for _, col := range usedCols {
 		idx := schema.ColumnIndex(col)
 		if idx == -1 {
-			log.Errorf("Can't find column %s from schema %s.", col, schema)
+			panic(fmt.Sprintf("Can't find column %s from schema %s.", col, schema))
```

Comment:
Why panic?

Reply:
Because if `idx == -1`, in line 40, `used[idx] = true` would panic also, to make the error message in log clearer, I made this change.